managed-ledger/src/main/java/org/apache/bookkeeper/mledger/util/RangeCache.java

```diff
@@ -90,7 +90,7 @@ public Value get(Key key) {
             try {
                 value.retain();
                 return value;
-            } catch (Throwable t) {
+            } catch (IllegalReferenceCountException e) {
```

Comment:
probably we should log something here.
this case must not happen

Reply:
yes, logging would be useful to get more information. I'm just wondering if it should be done only at debug level since it's not a real problem. It's part of the expected behavior that this could sometimes happen.